src/backend/vulkan/src/conv.rs

```diff
@@ -623,3 +623,103 @@ pub fn map_vk_memory_heap_flags(flags: vk::MemoryHeapFlags) -> hal::memory::Heap
 
     hal_flags
 }
+
+use hal::external_memory;
+
+pub fn map_external_memory_handle_types(
+    flags: external_memory::ExternalMemoryTypeFlags,
+) -> vk::ExternalMemoryHandleTypeFlags {
+    let mut vk_flags = vk::ExternalMemoryHandleTypeFlags::empty();
```

Comment:
if hal flags are 1:1 with Vulkan flags, we can just do `from_bits` without checking each individual bit

Reply:
Done!